### PR TITLE
Metrics wireformats

### PR DESCRIPTION
**Pull request by alesstimec**
- added metrics wireformats
- changes required by latest juju core master

### CODE REVIEW

**Comment by tasdomas** [comment]
LGTM

File: cmd/allocate/allocate_test.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func (s *allocateSuite) SetUpTest(c *gc.C) {
 func (s *allocateSuite) TestAllocate(c *gc.C) {
 	s.mockAPI.resp = "allocation updated"
 	alloc := allocate.NewAllocateCommand()
-	ctx, err := cmdtesting.RunCommand(c, alloc, "name:100", "db")
+	ctx, err := cmdtesting.RunCommand(c, alloc, "name:100", "db", "--model", coretesting.SampleModelName)
```
Comment (cmars) [inline cmd/allocate/allocate_test.go]: Nice catch!

**Comment by cmars** [comment]
LGTM!

**Comment by alesstimec** [comment]
$$merge$$

**Pull request merged**